### PR TITLE
Define scroll terminology

### PR DESCRIPTION
**Pull request by fantasai**
Addresses #8194 #7303 #7237 and does some additional editorial cleanup.

### CODE REVIEW

File: css-overflow-3/Overview.bs [hunk 1]
```diff
@@ -315,6 +321,90 @@ Scrollable Overflow</h3>
...
+	or by script
+	(for example, by the {{scrollIntoView()}} or {{focus()}} APIs).
+	The initial position of the [=scrollable overflow rectangle=] within the [=scrollport=]
+	before any scrolling operations take effect is the <dfn>initial scroll position</dfn>.
+	The [=initial scroll position=]
+	is typically dependent on the [=scroll container=]’s [=writing mode=],
+	and, unless otherwise specified,
+	coincides with its [=scroll origin position=].
+	However, the 'align-content' and 'justify-content' properties [[!CSS-ALIGN-3]]
+	can be used to change the [=initial scroll position=],
```
Comment (tabatkins) [inline css-overflow-3/Overview.bs:349]:
```suggestion
	can be used to change the [=initial scroll position=];
```

File: css-overflow-3/Overview.bs [hunk 2]
```diff
@@ -315,6 +321,90 @@ Scrollable Overflow</h3>
...
+	which is its distance from the [=scroll origin=].
+
+	The <dfn>scroll origin</dfn> is the anchor coordinate of the [=scrollable overflow rectangle=],
+	from which the [=scrollable overflow rectangle=] expands.
+	Unless otherwise specified,
+	it is the [=block-start=] [=inline-start=] corner of the [=scrollable overflow rectangle=].
+	(For example, in a [=flex container=] it is the [=main-start=] [=cross-start=] corner.)
+	The area beyond the [=scroll origin=] in either axis
+	is considered the <dfn>negative scrollable overflow region</dfn>:
+	content rendered here is the not accessible to the reader,
```
Comment (tabatkins) [inline css-overflow-3/Overview.bs:365]:
```suggestion
	content rendered here is not accessible to the reader,
```

**Review by tabatkins — APPROVED** [review]
Two suggested edits (just typo-fixing), otherwise LGTM

File: css-overflow-3/Overview.bs (hunk 1 above)
Comment (fantasai) [reply css-overflow-3/Overview.bs:349]: I think we usually use a comma in this pattern.

File: css-overflow-3/Overview.bs (hunk 2 above)
Comment (fantasai) [reply css-overflow-3/Overview.bs]: Fixed.

**Pull request merged**